test/highlight/filetype_statement.vim

```diff
@@ -0,0 +1,13 @@
+filetype
+" <- keyword
+
+filetype on
```

Comment:
Can you also check that the `on` is captued here too ?

Reply:
Sure, although I think that maybe `on` and `off` should be treated differently. But for now let's keep them as keywords as with `syntax`

Reply:
We _could_ mark them as `@boolean`, as this is close to their actual meaning.

Reply:
Yeah, it seems the best choice to me, but for consistency I think it would be better to handle it for both `syntax` and `filetype` (and any other command that uses `on`/`off`, if there is) in a separate PR, since this is already quite messy...